api/portainer.go

```diff
@@ -1393,7 +1393,7 @@ const (
 )
 
 // List of supported features
-var SupportedFeatureFlags = []Feature{}
+var SupportedFeatureFlags = []Feature{"dev-metrics"}
```

Comment:
Before removing the open-amt and fdo feature flags, these were defined using type Feature. I actually wouldn't mind using raw strings, but just raising this just in case
![image](https://user-images.githubusercontent.com/16310914/150791092-509d3544-8e28-4a20-82e9-0a1663385cd4.png)